students/meet_shah/lesson08/circle_test.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
```

Comment:
you need to make formal tests here!

example:
```
def test_diameter():
    c - Circle.from_diameter(10)
    assert c.diameter == 10
    assert c.radius == 5
```

and so on -- for all functionality.